### ISSUE TITLE
Question: Can I publish Pyflow on AUR?

### ISSUE THREAD

**Issue by ru5j4r0** [David-OConnor/pyflow#49]
First of all, thank you for your great work!
I really like its concept.

The Arch User Repositry (AUR) is a community-driven repository for Arch Linux users.
I'm using Manjaro Linux, the Arch based distro, so I made a AUR package for my own use.
Currently I'm using the package locally, but I interested to publish and maintain it.
If it's published, it'll be available through AUR helper like yay, tizen, etc.
Basically it's a kind of APT, Yum or Homebrew for Arch and Arch based distro users. (Technically it's a bit different though.)
Please let me know if it's ok with you.

**Comment by David-OConnor** [comment]
Yep. Let me know if there's anything you'd like me to do to help. I'll update the readme once complete.

**Comment by ru5j4r0** [comment]
Thanks for your support.
It's completed ([package page](https://aur.archlinux.org/packages/pyflow/)), and it's available through [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers) on Arch Linux and Arch based distro.
For example, if one using yay, Pyflow can be installed with `yay pyflow`.

Actually this is my first package maintaining, so sorry if there are any mistakes.
I will try to maintain it correctly.

Please let me know if there are needed informations for you to update the readme.

**Comment by David-OConnor** [comment]
Nice!
